services/thumbnails/README.md

```diff
@@ -0,0 +1,29 @@
+# Thumbnails service
+
+The thumbnails service provides methods to generate thumbnails for various files.
+It retrieves the sources at the location where the user files are stored and saves the tumbnails at the location where system files are stored. Those locations have defaults but can be manually defined via environment variables. The relevant environment variables are:
+
+-   `OCIS_BASE_DATA_PATH` which will contain system relevant data and
+-   `STORAGE_USERS_OCIS_ROOT` which will contain the user source data.
+-   `THUMBNAILS_FILESYSTEMSTORAGE_ROOT` used if thumnails should be separated from system files.
+
+For details and defaults see the documentation.
+
+Thumbnails can be generated for the following file types:
```

Comment:
Q1: are we identifying the image type by extension or by magic bytes or both?
Q2: If the image type is identified by extension, what happens if the file is not of the type?

Reply:
> Are we identifying the source image type by extension or by magic bytes or both?

The thumbnail service is just using the information given by the backend. The backend is identifying it based on the extension.

> If the source image type is identified by extension, what happens if the file is not of the type? Means extension ≠ magic bytes?

Thumbnail generation will fail.